srcbook/src/server.mjs

```diff
@@ -64,7 +64,6 @@ server.listen(port, () => {
 });
 
 process.on('SIGINT', async () => {
-  await posthog.shutdown();
```

Comment:
Why are you removing this?

Reply:
Didn't see the call to the underlying posthog shutdown